test/algorithms/afl/loop.cpp

```diff
@@ -53,42 +52,17 @@ static void AFLLoop(bool forksrv) {
   auto input_dir = put_dir / "seeds";
   auto output_dir = root_dir / "output";
 
-  // Create fuzzer instance
-  // FIXME: we can use BuildAFLFuzzerFromArgs after it supports forksrv as an
-  // option
-  using namespace fuzzuf::algorithm::afl;
-  using fuzzuf::executor::AFLExecutorInterface;
-  using option::AFLTag;
+  auto input_dir_opt  =  "--in_dir=" +  input_dir.string();
+  auto output_dir_opt = "--out_dir=" + output_dir.string();
+  const char *forksrv_opt = forksrv ? "--forksrv=1" : "--forksrv=0";
 
-  std::shared_ptr<AFLSetting> setting(new AFLSetting(
-      {"../../put_binaries/libjpeg/libjpeg_turbo_fuzzer", "@@"},
-      input_dir.native(), output_dir.native(), option::GetExecTimeout<AFLTag>(),
-      option::GetMemLimit<AFLTag>(), forksrv, false, /* dumb_mode*/
-      Util::CPUID_BIND_WHICHEVER));
-
-  // NativeLinuxExecutor needs the directory specified by "out_dir" to be
-  // already set up so we need to create the directory first, and then
-  // initialize Executor
-  SetupDirs(setting->out_dir.string());
-
-  // Create NativeLinuxExecutor
-  auto nle = std::make_shared<NativeLinuxExecutor>(
-      setting->argv, setting->exec_timelimit_ms, setting->exec_memlimit,
-      setting->forksrv, setting->out_dir / option::GetDefaultOutfile<AFLTag>(),
-      option::GetMapSize<AFLTag>(), // afl_shm_size
-      0                             // bb_shm_size
-      );
-
-  auto executor = std::make_shared<AFLExecutorInterface>(std::move(nle));
-
-  // Create AFLState
-  auto state = std::make_unique<AFLState>(setting, executor);
-
-  AFLFuzzer fuzzer(std::move(state));
+  int argc = 7;
+  const char *argv[] = {"fuzzuf", "afl", input_dir_opt.c_str(), output_dir_opt.c_str(), forksrv_opt, "../../put_binaries/libjpeg/libjpeg_turbo_fuzzer", "@@"};
```

Comment:
```suggestion
  const char *argv[] = {"fuzzuf", "afl", input_dir_opt.c_str(), output_dir_opt.c_str(), forksrv_opt, "../../put_binaries/libjpeg/libjpeg_turbo_fuzzer", "@@"};
  int argc = static_cast<int>(sizeof(argv) / sizeof(const char *));
```
Why not dynamically set `argc` instead of hardcoding? 
I also thought of terminating `argv` with `nullptr`, but boost::program_options will handle it nicely?

Reply:
Oh absolutely you are right. 

I think boost::program_options actually should work without `nullptr`, considering that `argc` tells the number of elements in `argv` and that there is usually no need to access `argv[argc]`.
However, according to [the document of boost](https://www.boost.org/doc/libs/1_54_0/doc/html/boost/program_options/basic_command_line_parser.html), _the parameters should be the same as passed to 'main'._ So I must fix this.

Reply:
```suggestion
  const char *argv[] = {"fuzzuf", "afl", input_dir_opt.c_str(), output_dir_opt.c_str(), forksrv_opt, "../../put_binaries/libjpeg/libjpeg_turbo_fuzzer", "@@", nullptr};
  int argc = static_cast<int>(sizeof(argv) / sizeof(const char *));
```

Reply:
fixed in https://github.com/fuzzuf/fuzzuf/pull/65/commits/5a62cd348cf920b9764341b21d964ece255e8d65.
Not related to this PR, but the test of IJON is also affected. So I fixed it too.